### PR TITLE
ST-1229 Doc management changes

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
### Change description ###

Doc management changes

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ST-1229

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/sptribs/document/model/DocumentType.java [hunk 1]
```diff
@@ -12,87 +12,129 @@ public enum DocumentType implements HasLabel {
...
+    @JsonProperty("Expert evidence")
+    @JsonAlias({"C Docs"})
+    EXPERT_EVIDENCE("C - Expert evidence", "Expert evidence", "C"),
 
-    @JsonProperty("Statements")
-    @JsonAlias({"S Docs"})
-    STATEMENTS("Statements"),
+    @JsonProperty("Other medical records")
+    @JsonAlias({"C Docs"})
+    OTHER_MEDIAL_RECORDS("C - Other medical records", "Other medical records", "C"),
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/document/model/DocumentType.java:70]: Typo here - OTHER_MEDICAL_RECORDS?

File: src/main/java/uk/gov/hmcts/sptribs/document/model/DocumentType.java [hunk 2]
```diff
@@ -12,87 +12,129 @@ public enum DocumentType implements HasLabel {
...
+
+    @JsonProperty("Local Authority/care records")
+    @JsonAlias({"E Docs"})
+    LOCAL_AUTHORITY_CARE_RECORDS("E - Local Authority/care records", "Local Authority/care records", "E"),
 
-    @JsonProperty("nameChangeEvidence")
-    NAME_CHANGE_EVIDENCE("Name change evidence"),
+    @JsonProperty("Other-E")
+    @JsonAlias({"E Docs"})
+    OTHER_E("E - Other", "Other-E", "E"),
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/document/model/DocumentType.java:106]: Can we call this OTHER_EVIDENCE? "Other E" is a bit ambiguous

File: src/main/java/uk/gov/hmcts/sptribs/document/model/DocumentType.java [hunk 3]
```diff
@@ -12,87 +12,129 @@ public enum DocumentType implements HasLabel {
...
-    AOS_OVERDUE_LETTER("Aos overdue letter"),
+    @JsonProperty("Submission from respondent")
+    @JsonAlias({"TG Docs"})
+    SUBMISSION_FROM_RESPONDENT("TG - Submission from respondent", "Submission from respondent", "TG"),
 
-    @JsonProperty("generalLetter")
-    GENERAL_LETTER("General letter");
+    @JsonProperty("Other-TG")
+    @JsonAlias({"TG Docs"})
+    OTHER_TG("TG - Other", "Other-TG", "TG");
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/document/model/DocumentType.java:135]: Again, as above, expand "TG"?

**Review by osheaf01 — CHANGES_REQUESTED** [review]
There's a typo which must be fixed, and a couple of suggestions...

**Review by sjamihmcts — DISMISSED** [review]

**Review by osheaf01 — APPROVED** [review]
LGTM

**Review by osheaf01 — APPROVED** [review]
LGTM

**Pull request merged**